.github/workflows/root-ci.yml

```diff
@@ -118,6 +118,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.ref_name }}
```

Comment:
Why is the default not working?

Reply:
As the workflow is called from `master`, the `checkout` action would default to `master` for all nightly builds. However, we want the build configuration from the respective branches. (In principle it's not needed on `master` itself, but since we create release branches from it, it makes sense to change it there and not having to worry about it)

Reply:
Thanks for the clarification.  It was not clear from the description which use case (nighties) this PR was fixing.